src/actions/expressions.js

```diff
@@ -32,6 +32,13 @@ export function addExpression(input: string) {
       return;
     }
 
+    const error = await parser.hasSyntaxError(input);
+    if (error) {
+      // for debugging only
+      alert("'" + input + "' " + "has syntax error");
```

Comment:
lol - didn't realize this was a WIP pr

Reply:
Oops! I forgot to delete the comment lol